### DISCUSSION TITLE
A counter for the amount of legacy settings detected

### DISCUSSION THREAD

**Discussion by MysteriousOrb2000** [Docile-Alligator/Infinity-For-Reddit#876] [Ideas]
In Settings > Advanced > Delete All Legacy Settings, I was wondering if you add a count underneath that setting for the amount of legacy settings detected in the database. This would help notify the user that there were settings that were marked by the user within the database that no longer exists in the newer version of the app.

**Comment by Wladefant** [comment]
@MysteriousOrb2000 Do the legacy settings apply to the legacy mode? (Old reddit)

**Comment by MysteriousOrb2000** [comment]
I think it's mostly for the app itself rather than Reddit.
